javatests/arcs/core/crdt/Generators.kt

```diff
@@ -0,0 +1,65 @@
+package arcs.core.crdt
+
+import arcs.core.common.Referencable
+import arcs.core.data.RawEntity
+import arcs.core.testutil.Generator
+
+/**
+ * Generate a [CrdtEntity] given a generator for the [VersionMap] and [RawEntity].
+ */
+class CrdtEntityGenerator(
+  val version: Generator<VersionMap>,
+  val rawEntity: Generator<RawEntity>
+) : Generator<CrdtEntity> {
+  override operator fun invoke(): CrdtEntity {
+    return CrdtEntity(
+      version(),
+      rawEntity()
+    )
+  }
+}
+
+/**
+ * Generate a [VersionMap] given generators for the actor and version.
+ */
+class VersionMapGenerator(
```

Comment:
This only creates VersionMaps with a single actor, is that correct? Make this clear in the comment and by calling it SingleActorVersionMapGenerator